publish.go

```diff
@@ -122,15 +122,15 @@ func sshSession() (*ssh.Session, error) {
 }
 
 func publishShareInstructions(url string) {
-	fmt.Println("\n" + GrayStyle.Render("  Share your GIF with Markdown:"))
```

Comment:
Need to check if this breaks `vhs publish` as it depends on outputting the shareable URL

Reply:
The logger doesn't get initialized at the beginning of the `publish` subcommand, so just PR'd your branch to correct that :) Looks great!